db-scripts/src/main/resources/cgds.sql

```diff
@@ -775,6 +791,22 @@ CREATE TABLE `clinical_event_data` (
 );
 
 -- --------------------------------------------------------
+DROP TABLE IF EXISTS `reference_genome_gene`;
+CREATE TABLE `reference_genome_gene` (
+    `ENTREZ_GENE_ID` int(11) NOT NULL,
+    `CYTOBAND` varchar(64) NOT NULL,
+    `EXONIC_LENGTH` int(11) DEFAULT NULL,
```

Comment:
Is length in the gene table going to be removed in a future PR?

Reply:
yes, that is the plan.